src/main/kotlin/no/nav/familie/ef/sak/vedtak/VedtakService.kt

```diff
@@ -100,6 +119,62 @@ class VedtakService(
     fun hentHarAktivtVedtak(behandlingId: UUID, localDate: LocalDate = LocalDate.now()): Boolean {
         return hentVedtak(behandlingId).erVedtakAktivtForDato(localDate)
     }
+
+    @Transactional
```

Comment:
Må vi sende noe till behandlingsstatistikk her?

Reply:
Virker ikke som at det gjøres når et vedtak underkjennes

Reply:
Tenker vi ikke trenger det, når vi heller ikke gjør det for behandling underkjent.